tdrs-backend/.env.example

```diff
@@ -0,0 +1,38 @@
+### Local Development Environment
+# Copy this file to `.env` and replace variables as needed
+#
+DISABLE_COLLECTSTATIC=1
+
+# Parameters defined in the login.gov documentation https://developers.login.gov/oidc/
+CLIENT_ID=urn:gov:gsa:openidconnect.profiles:sp:sso:hhs:tanf-proto-dev
+OIDC_RP_CLIENT_ID=urn:gov:gsa:openidconnect.profiles:sp:sso:hhs:tanf-proto-dev
+ACR_VALUES=http://idmanagement.gov/ns/assurance/ial/1
+OIDC_OP_ISSUER=https://idp.int.identitysandbox.gov/
+OIDC_OP_JWKS_ENDPOINT=https://idp.int.identitysandbox.gov/api/openid_connect/certs
+OIDC_OP_AUTHORIZATION_ENDPOINT=https://idp.int.identitysandbox.gov/openid_connect/authorize
+OIDC_OP_LOGOUT_ENDPOINT=https://idp.int.identitysandbox.gov/openid_connect/logout
+OIDC_OP_TOKEN_ENDPOINT=https://idp.int.identitysandbox.gov/api/openid_connect/token
+OIDC_OP_USER_ENDPOINT=https://idp.int.identitysandbox.gov/api/openid_connect/userinfo
+CLIENT_ASSERTION_TYPE=urn:ietf:params:oauth:client-assertion-type:jwt-bearer
+
+# Local django settings to define the execution environment
+DJANGO_SETTINGS_MODULE=tdpservice.settings.local
+DJANGO_CONFIGURATION=Local
+
+# Sets the URL values used for redirects
+FRONTEND_BASE_URL=http://localhost:3000
+BASE_URL=http://localhost:8080/v1
+
+# mock token for unit testing
+# saving here to resolve linting error for string too long
+MOCK_TOKEN=eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJzdWIiOiJiMmQyZDExNS0xZDdlLTQ1NzktYjlkNi1mOGU4NGY0ZjU2Y2EiLCJpc3MiOiJodHRwczovL2lkcC5pbnQubG9naW4uZ292IiwiYWNyIjoiaHR0cDovL2lkbWFuYWdlbWVudC5nb3YvbnMvYXNzdXJhbmNlL2xvYS8xIiwibm9uY2UiOiJhYWQwYWE5NjljMTU2YjJkZmE2ODVmODg1ZmFjNzA4MyIsImF1ZCI6InVybjpnb3Y6Z3NhOm9wZW5pZGNvbm5lY3Q6ZGV2ZWxvcG1lbnQiLCJqdGkiOiJqQzdOblU4ZE5OVjVsaXNRQm0xanRBIiwiYXRfaGFzaCI6InRsTmJpcXIxTHIyWWNOUkdqendsSWciLCJjX2hhc2giOiJoWGpxN2tPcnRRS196YV82dE9OeGN3IiwiZXhwIjoxNDg5Njk0MTk2LCJpYXQiOjE0ODk2OTQxOTgsIm5iZiI6MTQ4OTY5NDE5OH0.pVbPF-2LJSG1fE9thn27PwmDlNdlc3mEm7fFxb8ZADdRvYmDMnDPuZ3TGHl0ttK78H8NH7rBpH85LZzRNtCcWjS7QcycXHMn00Cuq_Bpbn7NRdf3ktxkBrpqyzIArLezVJJVXn2EeykXMvzlO-fJ7CaDUaJMqkDhKOK6caRYePBLbZJFl0Ri25bqXugguAYTyX9HACaxMNFtQOwmUCVVr6WYL1AMV5WmaswZtdE8POxYdhzwj777rkgSg555GoBDZy3MetapbT0csSWqVJ13skWTXBRrOiQQ70wzHAu_3ktBDXNoLx4kG1fr1BiMEbHjKsHs14X8LCBcIMdt49hIZg
+
+DJANGO_SECRET_KEY=w0&df2562wx3g9(cddtwtq)s*3k5jyza8*xwka$+=ya21h@#b(
```

Comment:
I think adding an inline note about `DJANGO_SECRET_KEY` would be helpful. It wasn't clear to me if I should be changing this to something else in my local setup.

Reply:
This is only secret in production and will run with any value locally - I will update it to something that looks less "secrety"

Reply:
@hursey013 this has been updated accordingly

Reply:
Thanks!